apollo-router/tests/integration/fixtures/query_planner_redis_config_update_query_fragments.router.yaml

```diff
@@ -7,5 +7,5 @@ supergraph:
         urls:
           - redis://localhost:6379
         ttl: 10s
-  generate_query_fragments: true
+  generate_query_fragments: false
```

Comment:
was this intentional switch to false? guess the test is to verify non-default?

Reply:
Yes, it's to make sure the cache key updates when the config is flipped